vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import pandas
+
+
+# Function to create model
+def emergent_object_model(path_to_weights):
```

Comment:
Also, will need to add type hints for parameter/return types: https://missourimrr.github.io/docs/python/type-annotations/full-guide.html